portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,19 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Aston Yong's Portfolio</h1>
```

Comment:
It looks like the formatting on this page is a little off. Try copy pasting this into a formatter:

https://www.freeformatter.com/html-formatter.html#ad-output

and use that instead.